docs/api/buckets.rst

```diff
@@ -3,8 +3,7 @@
 Buckets
 #######
 
-A bucket is the parent object of collections and groups. It can be viewed as
-a namespace where all collections and groups are stored.
+A bucket is the parent object of collections and groups.
```

Comment:
Why removing this information? When explaining what a bucket is, I find it very useful to say it's a namespace. Even yesterday, when working with Quentin, the first thing he said was "oh, okay, it's a namespace", without me saying it.